test/integration/krane_test.rb

```diff
@@ -16,6 +16,15 @@ def test_restart_black_box
     refute(fetch_restarted_at("redis"), "RESTARTED_AT env is present")
   end
 
+  def test_deploy_black_box
+    setup_template_dir("hello-cloud", subset: ["configmap-data.yml", "web.yml.erb", "redis.yml"]) do |target_dir|
```

Comment:
why not everything in hello-cloud?

Reply:
Just to simplify, really. If we use everything in hello-cloud we also need to specify some bindings (for the `.erb` files there). Do you think it would be better that way?

Reply:
I think we should test all of hello-cloud